backend/src/main/java/heartbeat/controller/pipeline/PipelineController.java

```diff
@@ -24,8 +27,10 @@ public class PipelineController {
 	private final BuildKiteService buildKiteService;
 
 	@GetMapping("/{pipelineType}")
-	public BuildKiteResponse getBuildKiteInfo(@PathVariable String pipelineType) {
-		return buildKiteService.fetchPipelineInfo();
+	public BuildKiteResponse getBuildKiteInfo(@PathVariable String pipelineType,
+			@RequestHeader("Authorization") @NotBlank(message = "Token must not be blank") String token,
+			@RequestParam String startTime, @RequestParam String endTime) throws NoPermissionException {
```

Comment:
Is the app handling the `NoPermissionException`?

Reply:
done

Reply:
Should be http code 401